app/src/utils/constants/application.ts

```diff
@@ -19,3 +19,6 @@ export const GROUP_NAME_LIST = [
 export const YES_NO_LIST = [BasicResponse.YES, BasicResponse.NO];
 
 export const YES_NO_UNSURE_LIST = [BasicResponse.YES, BasicResponse.NO, BasicResponse.UNSURE];
+
+/** Minimal viable UUIDv4 for the system ID */
```

Comment:
Add reference documents and citations where possible so that it is easier to look up background reasons.
```ts
/** 
  * Minimally valid UUIDv4 for the system ID
  * @see https://www.rfc-editor.org/rfc/rfc9562#variant_field
  */
```